src/config/section/image.js

```diff
@@ -43,7 +43,15 @@ export default {
         }
         return fields
       },
-      details: ['name', 'id', 'displaytext', 'checksum', 'hypervisor', 'format', 'ostypename', 'size', 'isready', 'passwordenabled', 'directdownload', 'deployasis', 'isextractable', 'isdynamicallyscalable', 'ispublic', 'isfeatured', 'crosszones', 'type', 'account', 'domain', 'created', 'url'],
+      details: () => {
+        var fields = ['name', 'id', 'displaytext', 'checksum', 'hypervisor', 'format', 'ostypename', 'size', 'isready', 'passwordenabled',
+          'directdownload', 'deployasis', 'isextractable', 'isdynamicallyscalable', 'crosszones', 'type',
+          'account', 'domain', 'created']
+        if (['Admin'].includes(store.getters.userInfo.roletype)) {
+          fields.push('ispublic', 'isrouting', 'isfeatured', 'templatetype', 'url')
```

Comment:
@ravening cc @weizhouapache @DaanHoogland can you check if this causes regression wrt legacy UI? The is public/routing/features and `url` were previously accepted by old UI as well. What about upload template API/form?

Reply:
`ispublic` and `isfeatured` is displayed for regular users also. so made the change